### PR TITLE
Stimpack Crafting Nerfs Ft. Legion Chems

### PR DESCRIPTION
**Pull request by Kavrick**
Made it so you can only make imitation stimpacks (because lore wise xander and broc _are_ what you make imitation with. 

The main reason behind this is that people just cheese literally the best healing item in the game. Stimpacks surpass all advanced chems by nearly five times, superstimpacks even more than this. The fact that the best healing chem bar none can be made en masse by planting three crops and printing a bunch of injectors is kinda a joke.

Also removed hand-crafting superstims because, hand-crafting the best pre-war medicine ever created using some flowers and some leather is ???

On top of that, the chemistry recipe for superstims replaced the muttfruit juice ingredient with plasma, making it so you actually have to put in effort in order to make such a rediculously strong healing item.

### CODE REVIEW

File: code/datums/components/crafting/recipes/recipes_medicine.dm [hunk 1]
```diff
@@ -132,34 +132,34 @@
...
-/datum/crafting_recipe/superstimpak5
-	name = "Super Stimpak (x5)"
-	result = /obj/item/storage/box/medicine/stimpaks/superstimpaks5
-	reqs = list(/obj/item/reagent_containers/hypospray/medipen/stimpak = 5,
-				/obj/item/stack/sheet/leather = 10,
-				/obj/item/reagent_containers/food/snacks/grown/mutfruit = 10)
-	tools = list(TOOL_WORKBENCH)
-	time = 30
-	category = CAT_MEDICAL
+//datum/crafting_recipe/superstimpak
```
Comment (ProbablyCarl) [inline code/datums/components/crafting/recipes/recipes_medicine.dm:144]: Open and close comments with /* and */, respectively. Don't use // with larger comments.

File: code/datums/components/crafting/recipes/recipes_medicine.dm [hunk 2]
```diff
@@ -132,34 +132,34 @@
...
 	tools = list(TOOL_WORKBENCH)
 	time = 20
 	category = CAT_MEDICAL
 
-/datum/crafting_recipe/superstimpak
-	name = "Super Stimpak"
-	result = /obj/item/reagent_containers/hypospray/medipen/stimpak/super
-	reqs = list(/obj/item/reagent_containers/hypospray/medipen/stimpak = 1,
+/*datum/crafting_recipe/superstimpak
+	/name = "Super Stimpak"
```
Comment (ProbablyCarl) [inline code/datums/components/crafting/recipes/recipes_medicine.dm:145]: /?

**Comment by UntoldTactics** [comment]
This is all good and dandy but what about bitterdrink and poultice? The level of effort between the two are debatable for sure.

**Comment by ExoticJazz** [comment]
I agree with the previous comment, if NCR are getting a nerf to their main source of healing; the Legion side needs to be looked at as well otherwise they will **heavily** benefit from this.

**Comment by Kavrick** [comment]
> 

I'm completely fine with holding off on merging this until that is sorted, i've been trying to think of a good compromise. My original idea is to nerf bitters down to an imitation stimpack level and then make hydra have a strong healing component, being a good, strong chem that is rarely used, but it should also see more use with the removal of wound armor. The other option was to make bitters harder to make. The reason behind this would be that nerfing bitters would basically just make it another healing powder, and generally you dont want multiple chems in the same niche.

**Comment by RadiantFlash7** [comment]
Lore wise it sucks yeah. But in this case, I almost want to say gameplay wise it doesn't?
Crafting stims and superstims quickly isn't horrible. and You can surpass it's healing power with custom chemical mixes of combined chemicals.
Everything is pretty much based around healing in combat as of current, I'm not sure making healing items harder/more of a pain is the way to go.

**Comment by Kavrick** [comment]
> 

Making it so you constantly have access to the best healing item in the game is bad, it ruins variety and breaks balance. People literally only have this opinion because it became the norm, no offence.

**Comment by Kavrick** [comment]
So, after the talks of making sure the legion stuff is on a similar level.

I nerfed all tribal healing chems across the board, with healing powder being slightly stronger than omnizine for comparison (healing powder is 1 heal per tick if you're tribal, omni is 0.6) 

To make up for this, I added an incredibly strong healing component to hydra, which already also heals wounds. 8 heal per tick to all damage types if you're tribal (or legion)

**Comment by ExoticJazz** [comment]
Looks good to me, will let @ProbablyCarl finalize this since medicine always heavily affects combat balance.

**Comment by projectkepler-ru** [comment]
why

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 3]
```diff
@@ -395,11 +395,11 @@
 	name = "Super Stimpak Fluid"
 	id = /datum/reagent/medicine/super_stimpak
 	results = list(/datum/reagent/medicine/super_stimpak = 2)
-	required_reagents = list(/datum/reagent/blood = 1, /datum/reagent/medicine/stimpak = 1, /datum/reagent/consumable/mutjuice = 1) //1 mutfruit at 50 potency yields 6 mutfruit juice
+	required_reagents = list(/datum/reagent/blood = 1, /datum/reagent/medicine/stimpak = 1, /datum/reagent/toxin/plasma = 1) 
 
 /datum/chemical_reaction/superstimpak/synthetic
 	id = "super_stimpak_synthetic"
-	required_reagents = list(/datum/reagent/blood/synthetics = 1, /datum/reagent/medicine/stimpak = 1, /datum/reagent/consumable/mutjuice = 1) //1 mutfruit at 50 potency yields 6 mutfruit juice
+	required_reagents = list(/datum/reagent/blood/synthetics = 1, /datum/reagent/medicine/stimpak = 1, /datum/reagent/toxin/plasma = 1) //1 mutfruit at 50 potency yields 6 mutfruit juice
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/chemistry/recipes/medicine.dm:402]:
```suggestion
	required_reagents = list(/datum/reagent/blood/synthetics = 1, /datum/reagent/medicine/stimpak = 1, /datum/reagent/toxin/plasma = 1)
```

**Comment by RadiantFlash7** [comment]
I'm not really sure how this accomplishes much, to be quite honest. You nerfed tribal chems because they were easily obtainable, and swapped out stimpaks for imitation stims. 

But since the recipe for Stimpak fluid has stayed the same, It's litterally easy as hell to mass produce stims in chemistry? It'll make anyone who doesn't have the chemist perk have less decent stimpaks, But this still keeps the NCR and followers able to easily mass produce said super stims?

Superstims were already better at healing then bitters. I'm not sure how to feel about this change.
I guess it would make hydra the new bitters? but- It's... not even that harder to get rad scorpion meat for it. Hell, You can probably get it faster then you would making bitters. But the real concern of this is, theres a destinct limit to the amount of rad scorpion meat you can obtain on the map.

I don't know man, this seems to change a lot and doesn't really accomplish what you want.

**Comment by DriftingSkies** [comment]
Right, I'm asking here and now.

Why are we fixing something that wasn't inherently broken? Why are we touching something that has worked for how long now because of... What, lore?

We're mixing something up that doesn't need to be mixed up in my most humble opinion. I don't see the point in this Nerf other than "We're doing it because we want to, lol"

**Comment by Kavrick** [comment]
> We're mixing something up that doesn't need to be mixed up in my most humble opinion. I don't see the point in this Nerf other than "We're doing it because we want to, lol"

I explained why in the original post, if you're going to completely miss that and think it's  'just because we want to' then I cant take your comment in anything but bad faith.

**Comment by Kavrick** [comment]
Closing this based on a discussion on discord, dont say I never take feedback :^) a new pr looking at chem stuff will be around, Lyro might be checking out stimpacks

**Pull request closed without merging**